### PR TITLE
Fix LED bar flickers

### PR DESCRIPTION
**Pull request by pnt325**

### CODE REVIEW

File: examples/OneOpenAir/OneOpenAir.ino [hunk 1]
```diff
@@ -843,15 +840,42 @@ static void configUpdateHandle() {
...
+      }
+      ag->ledBar.show();
+      Serial.println("Set 'ledBarBrightness' brightness: " +
                      String(configuration.getLedBarBrightness()));
     }
+
+    if (configuration.isLedBarModeChanged()) {
+      Serial.println("Set 'ledBarMode' " + String(configuration.getLedBarMode()));
+      Serial.println("Get 'ledBarBrightness' " + String(configuration.getLedBarBrightness()));
+      if (configuration.getLedBarBrightness() == 0) {
```
Comment (nick-4711) [inline examples/OneOpenAir/OneOpenAir.ino:859]: I think the code is redundant to what we have on line 843 onwards?

Comment (pnt325) [reply examples/OneOpenAir/OneOpenAir.ino:859]: Yes, I make it for test. We can remove it.

File: examples/OneOpenAir/OneOpenAir.ino [hunk 2]
```diff
@@ -843,15 +840,42 @@ static void configUpdateHandle() {
...
     if (configuration.isLedBarBrightnessChanged()) {
-      ag->ledBar.setBrighness(configuration.getLedBarBrightness());
-      Serial.println("Set 'LedBarBrightness' brightness: " +
+      if (configuration.getLedBarBrightness() == 0) {
+        ag->ledBar.setEnable(false);
+      } else {
+        if (configuration.getLedBarMode() != LedBarMode::LedBarModeOff) {
+          ag->ledBar.setEnable(true);
+        }
+        ag->ledBar.setBrighness(configuration.getLedBarBrightness());
```
Comment (nick-4711) [inline examples/OneOpenAir/OneOpenAir.ino:849]: setBrighness -> setBrightness

**Review by nick-4711 — APPROVED** [review]

File: examples/OneOpenAir/OneOpenAir.ino [hunk 3]
```diff
@@ -843,15 +840,42 @@ static void configUpdateHandle() {
...
+        }
+        ag->ledBar.setBrightness(configuration.getLedBarBrightness());
+      }
+      ag->ledBar.show();
+      Serial.println("Set 'ledBarBrightness' brightness: " +
                      String(configuration.getLedBarBrightness()));
     }
+
+    if (configuration.isLedBarModeChanged()) {
+      Serial.println("Set 'ledBarMode' " + String(configuration.getLedBarMode()));
```
[the remark this answers is not in the document]
Comment (pnt325) [reply examples/OneOpenAir/OneOpenAir.ino:857]: Ok, I'll remove it. It's just for test.

**Pull request merged**